.rubocop.yml

```diff
@@ -20,3 +20,7 @@ Rails:
 
 Style/FrozenStringLiteralComment:
   Enabled: false
+
+Metrics/MethodLength:
+  Exclude:
+    - 'spec/requests/*'
```

Comment:
I'll leave it up to you. Not sure any real value in listing each file, especially as chances are you will need to list ever file.